pom.xml

```diff
@@ -9,6 +9,7 @@
 	</parent>
 	<artifactId>bankrupt-officer-search-api</artifactId>
 	<version>unversioned</version>
+    <packaging>jar</packaging>
```

Comment:
Could you please align the formatting of this line?

Reply:
It is aligned within the IDE: 

![Screen Shot 2021-03-01 at 09 53 00](https://user-images.githubusercontent.com/55084241/109480889-0397b100-7a74-11eb-84a2-01f4a1edbde3.png)

Reply:
I have reformatted the POM in line with the CH guidelines